src/user.c

```diff
@@ -504,7 +504,7 @@ void sync_cflag(const Cflag *cflag)
     Nick *nptr;
     const char *bot;
 
-    if ((nptr = get_core_api()->find_nick(cflag->user->nick)) == NULL)
+    if ((nptr = get_core_api()->find_nick(cflag->user->authed_nick)) == NULL)
```

Comment:
user->authed_nick is a struct nick *, while find_nick() takes a const char *. I expect this is generating a compilation warning. A side effect is that find_nick() will always return NULL or segfault, so the code below is never executed.